### ISSUE TITLE
Sources expandeable but not hidden

### ISSUE THREAD

**Issue by loleg** [cividi/spatial-data-package-platform#30]
I can understand the UX decision to try to minimize the amount of space taken up by the source links, but they are one of the most important features of any data portal - and we have already received some community feedback about this bit of the interface. I would suggest that you see the first line and click "more..." or so, but at least one line of sources is always visible. Alternatively, the sources could be shown using standard control overlaid on the map to the left of the legend.

**Comment by n0rdlicht** [comment]
Agreed. We will need to find a way to add the OpenStreetMap attribution to empty Gemeinden since those do not have any sources at the moment. Maybe by converting them (see #29) to actual or virtual snapshots with an info box would help here.

I would prefer showing the first entry and (if available) a more button like the legend.
